star-navigation/src/components/button.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { clsx } from "clsx";
+
+import css from "./button.scss";
+
+interface IProps {
+  children: React.ReactNode;
+  className?: string;
+  onClick?: () => void;
+}
+
+export const Button: React.FC<IProps> = ({ children, className, onClick }) => {
```

Comment:
fyi: I think `React.FC` automatically adds the `children` to the props so you don't need to do it explicitly in the `IProps` interface.

Reply:
@dougmartin, it used to, but not anymore in React v18. It's one of the breaking changes.

Reply:
@pjanik Thanks - good to know.